docs/api/item.md

```diff
@@ -28,6 +32,125 @@ import Basic from '@site/static/usage/v7/item/basic/index.md';
+
+import Metadata from '@site/static/usage/v7/item/content-types/metadata/index.md';
+
+<Metadata />
+
+### Actions
+
+Actions are interactive elements that do something when you activate them. An item can have multiple actions displayed on a line. However, developers should ensure that each action's tap target is large enough to be usable.
+
+<BestPracticeFigure 
+  text={<>Developers should avoid creating <a href="https://dequeuniversity.com/rules/axe/4.4/nested-interactive">nested interactives</a> which can break the user experience with screen readers. For example, developers should avoid adding a button inside the main content of the Item if the <code>button</code> property is set to <code>true</code>. Additionally, developers should use the <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a> component if they need to show more than two actions.</>}
+  doText={<>Avoid nested interactives and put multiple items in <a href={useBaseUrl('api/item-sliding')}>Item Sliding</a> that can be revealed by swiping on the Item.</>}
+  dontText="Don't put too many actions on a single item."
+  doImage={<img alt="A list that contains several items, each representing a contact. Each item has text that states the contact's name as well as several actions including pinning the contact, sharing the contact, and deleting the contact. These actions are revealed by swiping on the item." src={useBaseUrl('img/item/actions-do.jpg')} />}
+  dontImage={<img alt="A list that contains several items, each representing a contact. Each item has text that states the contact's name as well as several actions including pinning the contact, sharing the contact, and deleting the contact. The actions are placed directly on the item. Since there are so many actions, some of the text is cut off." src={useBaseUrl('img/item/actions-dont.jpg')} />}
+/>
+
+In the example below, we are creating a list of contacts. Each item is a stubbed button intended to bring you to the full contact page for that item. There are additional actions associated with each item that users can reveal by swiping on the item.
+
+import Actions from '@site/static/usage/v7/item/content-types/actions/index.md';
+
+<Actions />
+
+### Controls
+
+Controls are form components such as checkboxes, inputs, radios, and more. Each item in a list should have at most two controls due to screen space constraints.
+
+<BestPracticeFigure 
+  text={<>Metadata such as helper text or character counts should not be used on form controls in list views. If such metadata is needed, the form control should be placed outside of a list. Alternatively, the context can be placed in a <a href="useBaseUrl('api/note')">Note</a> at the bottom of the list.</>}
```

Comment:
```suggestion
  text={<>Metadata such as helper text or character counts should not be used on form controls in list views. If such metadata is needed, the metadata should be placed outside of a list. Alternatively, the helper text can be placed in a <a href="useBaseUrl('api/note')">Note</a> at the bottom of the list.</>}
```

Shouldn't this be the metadata out of the list not the form control?

Reply:
In this case the helper text or characters counts refers to properties on `ion-input` and `ion-textarea`: https://ionicframework.com/docs/api/input#helper--error-text

You'd need to move the entire input in order to continue using this metadata

Reply:
Oh. It's a bit confusing with the image that follows. I expected it to be related to moving the text that says `"Password must be at least 16 characters."` outside of the list. Should we change the `"Do"` to portray what you are saying then?

Reply:
Developers have two options for using metadata with form controls:

1. Keep the control in the list, but place the metadata at the bottom of the list.
2. Move the control outside of the list.


We have an example of 1 but not of item 2. I'll work on adding item 2.

Reply:
Added in https://github.com/ionic-team/ionic-docs/pull/3123/commits/c86c8873abbff5f56d61380b24d19b6bb89c0ff5